merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/InsertSubsystemsAction.java

```diff
@@ -26,7 +26,7 @@
   private final PreparedStatement findStatement;
 
 
-  public InsertSubsystemsAction(final Connection connection) throws SQLException {
+  public  InsertSubsystemsAction(final Connection connection) throws SQLException {
```

Comment:
Minor, non-blocking: the whitespace in this file was accidentally changed:

```suggestion
  public InsertSubsystemsAction(final Connection connection) throws SQLException {
```